packages/block-editor/src/components/plain-text/index.js

```diff
@@ -22,13 +22,13 @@ const PlainText = forwardRef( ( { __experimentalVersion, ...props }, ref ) => {
 		return <EditableText ref={ ref } { ...props } />;
 	}
 
-	const { className, onChange, ...remainingProps } = props;
+	const { className, ...remainingProps } = props;
 
 	return (
-		<TextareaAutosize
+		<TextareaControl
 			ref={ ref }
 			className={ clsx( 'block-editor-plain-text', className ) }
-			onChange={ ( event ) => onChange( event.target.value ) }
```

Comment:
`TextareaControl`'s `onChange ` works directly with the value. No need to provide this, prop, it's passed down as part of `remainingProps` now.